libs/gi/db/src/Database/DataManagers/OptConfigDataManager.ts

```diff
@@ -200,27 +192,11 @@ export class OptConfigDataManager extends DataManager<
         .filter(([_, a]) => a.length)
     )
 
-    if (!Array.isArray(builds)) {
-      builds = []
-      buildDate = 0
-    } else {
-      builds = builds
-        .map((build) => {
-          if (typeof build !== 'object') return undefined
-          const { weaponId, artifactIds: artifactIdsRaw } =
-            build as GeneratedBuild
-          if (!this.database.weapons.get(weaponId)) return undefined
-          const artifactIds = objKeyMap(allArtifactSlotKeys, (slotKey) =>
-            this.database.arts.get(artifactIdsRaw[slotKey])?.slotKey === slotKey
-              ? artifactIdsRaw[slotKey]
-              : undefined
-          )
-
-          return { artifactIds, weaponId }
-        })
-        .filter((b) => b) as GeneratedBuild[]
-      if (!Number.isInteger(buildDate)) buildDate = 0
-    }
+    if (
+      generatedBuildListId &&
+      !this.database.generatedBuildList.get(generatedBuildListId)
+    )
+      generatedBuildListId = undefined
```

Comment:
(follow up to previous comment in `ArtCharDatabase.ts`) This will delete the generated list on initalization if the datamanagers are initialized in the wrong order.